src/prefect/agent/agent.py

```diff
@@ -38,10 +38,15 @@ class Agent:
     Args:
         - name (str, optional): An optional name to give this agent. Can also be set through
             the environment variable `PREFECT__CLOUD__AGENT__NAME`. Defaults to "agent"
+        - labels (List[str], optional): a list of labels, which are arbitrary string identifiers used by Prefect
+            Agents when polling for work
     """
 
-    def __init__(self, name: str = None) -> None:
+    def __init__(self, name: str = None, labels: Iterable[str] = None) -> None:
         self.name = name or config.cloud.agent.get("name", "agent")
+        self.labels = labels or config.cloud.agent.get("labels")
```

Comment:
have you tested this? I suspect you'll need to add:
```python
import ast

ast.literal_eval(config.cloud.agent.get("labels", 'None')) # note the quotes around None
```
but I'm not sure how smart our configuration object is when inferring lists from environment variables.